src/jit/morph.cpp

```diff
@@ -14057,6 +14057,12 @@ GenTree* Compiler::fgMorphModToSubMulDiv(GenTreeOp* tree)
         denominator = fgMakeMultiUse(&tree->gtOp2);
     }
 
+    if (lvaLocalVarRefCounted)
```

Comment:
> The returned values are used on line 14066 and 14068.

No, they are *cloned* there. As far as I can tell, the returned values are never used.

---
In reply to: [94656881](https://github.com/dotnet/coreclr/pull/8804#discussion_r94656881) [](ancestors = 94656881)